### PR TITLE
Bake direct eoc exercises

### PR DESCRIPTION
**Pull request by wilkus27**
In `pl-psychology` EOC exercises sections are direct children of chapter. (They're not inside composite chapter like in `u-physics` or `calculus`). 

I created two `erb` files: one for direct exercises section (`direct_review_exercises.xhtml.erb`) and one for exercises section from composite chapter (`child_review_exercises.xhtml.erb`). They differ only in two places, title tag and title from metadata:

image

Let me know what do you think about this solution and name convention of files.

### CODE REVIEW

File: lib/kitchen/directions/bake_chapter_review_exercises/v1.rb [hunk 1]
```diff
@@ -32,7 +32,13 @@ def bake(chapter:, metadata_source:, append_to:, klass:)
 
       @content = exercise_clipboard.paste
 
-      append_to.append(child: render(file: 'review_exercises.xhtml.erb'))
+      append_to_element = append_to || chapter
+
+      if append_to_element == append_to
+        append_to_element.append(child: render(file: 'child_review_exercises.xhtml.erb'))
+      elsif append_to_element == chapter
+        append_to_element.append(child: render(file: 'direct_review_exercises.xhtml.erb'))
```
Comment (LostMojo) [inline lib/kitchen/directions/bake_chapter_review_exercises/v1.rb:40]: Don't forget to write specs for this change!

File: lib/kitchen/directions/bake_chapter_review_exercises/direct_review_exercises.xhtml.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="os-eoc os-<%=@klass%>-container" data-type="composite-page" data-uuid-key=".<%=@klass%>">
+  <h2 data-type="document-title">
+    <span class="os-text"><%= @title %></span>
+  </h2>
+  <div data-type="metadata" style="display: none;">
+    <h1 data-type="document-title" itemprop="name"><%= @title %></h1>
```
Comment (LostMojo) [inline lib/kitchen/directions/bake_chapter_review_exercises/direct_review_exercises.xhtml.erb:6]: Will this work with internationalization?

**Review by LostMojo — CHANGES_REQUESTED** [review]
You've got an interesting solution! We might be able to cut it down to one base file if we just have a tag for h2/h3. 

Also, don't forget to write spec for your changes!

**Review by kswanson33 — COMMENTED** [review]
I like this solution as a way of managing the differences between books for the eoc exercises. I agree with @LostMojo -- I think we can use just one template file for both ways. An example would be something like:

```html
<div class="os-eoc os-<%=@klass%>-container" data-type="composite-page" data-uuid-key=".<%=@klass%>">
  <<%= @tag %> data-type="document-title">
    <span class="os-text"><%= @title %></span>
  </<%= @tag %>>
  <div data-type="metadata" style="display: none;">
    <h1 data-type="document-title" itemprop="name"><%= @metadata_title %></h1>
    <%= @metadata.paste %>
  </div>
  <%= @content %>
</div>
```

where `@tag` is either `h2` or `h3` and `@metadata_title` is either `@title` or `I18n.t(:eoc_exercises_title)`

File: lib/kitchen/directions/bake_chapter_review_exercises/v1.rb (hunk 1 above)
Comment (wilkus27) [reply lib/kitchen/directions/bake_chapter_review_exercises/v1.rb]: After applying solution suggested by Kendra there is no need for new specs. All tests passed.

File: lib/kitchen/directions/bake_chapter_review_exercises/direct_review_exercises.xhtml.erb (hunk 2 above)
Comment (wilkus27) [reply lib/kitchen/directions/bake_chapter_review_exercises/direct_review_exercises.xhtml.erb]: For now I just added this line to `bake` file:

image

And it's working. In the future I'm going to apply such solution: https://github.com/openstax/kitchen/issues/185.

**Comment by wilkus27** [comment]
@kswanson33 @LostMojo Thank you for all your comments :relaxed: I applied solution suggested by you. It's ready for rereview.

File: lib/kitchen/directions/bake_chapter_review_exercises/v1.rb [hunk 3]
```diff
@@ -32,7 +32,11 @@ def bake(chapter:, metadata_source:, append_to:, klass:)
 
       @content = exercise_clipboard.paste
 
-      append_to.append(child: render(file: 'review_exercises.xhtml.erb'))
+      append_to_element = append_to || chapter
+      @tag = append_to ? 'h3' : 'h2'
+      @metadata_title = append_to ? I18n.t(:eoc_exercises_title) : @title
```
Comment (kswanson33) [inline lib/kitchen/directions/bake_chapter_review_exercises/v1.rb:37]: Even though Codecov says it's covered, I notice that the specs for v1 & v2 only have an example of this method being called with an append_to element, not without, so if there's anything unexpected with the behavior around appending to chapter, the spec won't catch it. Could you add that example to [the specs](https://github.com/openstax/kitchen/tree/main/spec/directions/move_exercises_to_eoc)?

**Review by kswanson33 — CHANGES_REQUESTED** [review]
`BakeChapterReviewExercises` is being renamed to `MoveExercisesToEOC` to reduce confusion about exercise directions. I've made a [PR](https://github.com/openstax/kitchen/pull/195) for you; if this looks good, feel free to merge it into the current PR to take care of the renaming on this branch.

File: lib/kitchen/directions/bake_chapter_review_exercises/v1.rb (hunk 3 above)
Comment (kswanson33) [reply lib/kitchen/directions/bake_chapter_review_exercises/v1.rb:37]: unless I'm missing something...

Comment (wilkus27) [reply lib/kitchen/directions/bake_chapter_review_exercises/v1.rb:37]: Thank you for catching this :relaxed: I will try to add all instances to specs.

**Comment by wilkus27** [comment]
@LostMojo @kswanson33 I added specs for both versions. PR is ready for review. I'm new to creating rspecs, so I'm not sure if this is how it suppose to look :grin: Let me know if I did something wrong.

**Review by kswanson33 — APPROVED** [review]
This looks great (& should fix a problem I was having with my book)! Thank you!

**Review by LostMojo — APPROVED** [review]
Awesome! This looks great, and I can't wait to use it!

**Pull request merged**